absearch/server.py

```diff
@@ -37,6 +38,34 @@ def reload():
 gevent.signal(signal.SIGINT, close)
 
 
+class _Statsd(object):
+    def __init__(self, config):
+        if config.get('datadog', True):
```

Comment:
If by "compatible with statsd" you mean with a statsd daemon / server, yes. The wrapper here is done because the python clients have slightly different methods signatures